content/en/service_catalog/software_templates.md

```diff
@@ -23,8 +25,8 @@ Use [App Builder][2] to create dynamic, user-friendly forms to collect inputs fr
 ### Create software template workflows
 To use software templates in Datadog, create a Git repository with the desired template. You can start from scratch or use our quickstart blueprints to learn from an example.
 
-#### Start from a blueprint
-Navigate to [App Builder Blueprints][9] and select one of the following blueprints:
+#### Start from an example
+Navigate to [App Builder Blueprints][9] and select one of the following blueprints. These are examples of how to configure an App &/or Workflow to help you get started. You can make updates to the examples to fit your needs, such as configuring inputs, setting up integrations with source code management and/or cloud providers, and configuring permissions.
```

Comment:
```suggestion
Navigate to [App Builder Blueprints][9] and select one of the following blueprints. These are examples of how to configure an App or Workflow to help you get started. You can make updates to the examples to fit your needs, such as configuring inputs, setting up integrations with source code management or cloud providers, and configuring permissions.
```

I realize "and/or" is technically more accurate, but it often confuses people and we try not to use it. When it's important to communicate the "and/or", you can add an "or both" afterward, but in high-level descriptions like these, I don't think the plain "or" will mislead folks.